tests/specs/neogit/config_spec.lua

```diff
@@ -0,0 +1,544 @@
+local config = require("neogit.config")
+
+describe("Neogit config", function()
+  before_each(function()
+    config.values = config.get_default_values()
+  end)
```

Comment:
Can you add a newline between `6` & `7`?

Reply:
Stylua is removing the newline as a heads up. Do you want me to add a stylua ignore?

Reply:
Eh, nah. I don't always agree with stylua, but it's not usually worth it to fight. Not _this_ time.